build.gradle.kts

```diff
@@ -15,10 +15,13 @@ repositories {
 }
 
 dependencies {
+    implementation("org.antlr:antlr4:4.13.1")
     implementation("dev.openfga:openfga-sdk:0.4.2")
     implementation("org.dmfs:oauth2-essentials:0.22.1")
     implementation("org.dmfs:httpurlconnection-executor:1.22.1")
 
+    implementation(files("libs/java-0.0.1.jar"))
```

Comment:
Needs to be replaced with published library

Reply:
I think, once published language will be:
```
    implementation("dev.openfga:openfga-language:0.2.0")
```

but for now, can you add that in a comment? That this cannot be built without manually building language and providing the jar file.

And let's merge and tag

Reply:
Comment added.